kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/AbstractPulsarClient.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.streamnative.pulsar.handlers.kop;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+import java.io.Closeable;
+import java.util.function.Consumer;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.pulsar.broker.PulsarServerException;
+import org.apache.pulsar.broker.PulsarService;
+import org.apache.pulsar.client.api.AuthenticationFactory;
+import org.apache.pulsar.client.api.PulsarClientException;
+import org.apache.pulsar.client.impl.PulsarClientImpl;
+import org.apache.pulsar.client.impl.conf.ClientConfigurationData;
+
+
+/**
+ * The abstraction wrapper of {@link PulsarClientImpl}.
+ */
+@Slf4j
+@Getter
+public abstract class AbstractPulsarClient implements Closeable {
+
+    private final PulsarClientImpl pulsarClient;
+
+    public AbstractPulsarClient(@NonNull final PulsarClientImpl pulsarClient) {
+        this.pulsarClient = pulsarClient;
+    }
+
+    @Override
+    public void close() {
+        try {
+            pulsarClient.close();
+        } catch (PulsarClientException e) {
+            log.warn("Failed to close PulsarClient of LookupClient", e);
```

Comment:
Now it also may be `SystemTopicClient`

Reply:
Now I printed the actual class name here.